controllers.py

```diff
@@ -33,22 +33,28 @@
 from .common import db, session, T, cache, auth, signed_url
 from .settings import APP_FOLDER
 import os
+import datetime
 
 url_signer = URLSigner(session)
 
 
 @action('index')
 @action.uses('index.html', db)
 def index():
-
-    return dict()
+    # Section is for the searchBar component
+    userinput = request.params.get('userinput')
+    if userinput == "" or userinput is None:
+        results = db(db.observations_na).select(limitby=(0,10))
+    else:
+        results = db(db.observations_na.species_guess.contains(userinput, all=True)).select(limitby=(0,10))
```

Comment:
I wouldnt search solely off of species_guess as that is user inputted, so perhaps search off of scientific_name and ideally search off of multiple params: species_guess,scientific_name,common_name,iconic_taxon_name